util/src/main/java/com/farao_community/farao/util/SingleNetworkPool.java

```diff
@@ -23,7 +23,7 @@
  * while correctly handling setup and cleanup of variants
  * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
  */
-class SingleNetworkPool extends FaraoNetworkPool {
+class SingleNetworkPool extends MultipleNetworkPool {
```

Comment:
extends AbstractNetworkPool

Reply:
I actually wanted to override MultipleNetworkPool to avoid duplicate code

Reply:
I put the common code in AbstractNetworkPool and inherited it in SingleNetworkPool